src/app/lib/config/services/uzbek.ts

```diff
@@ -1,352 +1,382 @@
+        button: 'Қабул қилинг ва давом этинг',
       },
+    },
+    include: {
+      errorMessage:
+        'Sorry, we can’t display this part of the story on this lightweight mobile page.',
+      linkText: 'View the full version of the page to see all the content.',
+    },
+    topStoriesTitle: 'Бош мақола',
+    featuresAnalysisTitle: 'Муҳаррир танлови',
+    latestMediaTitle: 'Сўнгги',
+  },
+  mostRead: {
+    header: 'Энг кўп ўқилган',
+    lastUpdated: 'Сўнгги янгиланиш:',
+    numberOfItems: 5,
+    hasMostRead: true,
+  },
+  mostWatched: {
+    header: 'Энг кўп кўрилган',
+    numberOfItems: 5,
+    hasMostWatched: true,
+  },
+  footer: {
+    trustProjectLink: {
+      href: 'https://www.bbc.com/uzbek/institutional-50220995',
+      text: 'Нега сиз Би-би-сига ишонишингиз мумкин?',
+    },
+    externalLink: {
+      href: 'https://www.bbc.co.uk/editorialguidelines/guidance/feeds-and-links',
+      text: 'Ташқи линкларга бизнинг ёндашувимиз қандайлиги ҳақида маълумотга эга бўлинг',
+    },
+    links: [
       {
-        title: 'Дунё',
-        url: '/uzbek/topics/cl8l9mved19t',
+        href: 'https://www.bbc.com/uzbek/institutional-36824297',
+        text: 'Фойдаланиш шартлари',
       },
       {
-        title: 'Спорт',
-        url: '/uzbek/topics/cxnykykk1zkt',
+        href: 'https://www.bbc.com/uzbek/institutional-36824300',
+        text: 'Шахсий ҳаёт махфийлиги сиёсати',
       },
       {
-        title: 'Илм-Фан',
-        url: '/uzbek/topics/cg7262681krt',
+        href: 'https://www.bbc.com/usingthebbc/cookies/',
+        text: 'Куки(Cookies)',
       },
       {
-        title: 'Технология',
-        url: '/uzbek/topics/cjgn7n7v3yjt',
+        href: 'https://www.bbc.co.uk/uzbek/send/u50853929',
+        text: "'Bi-bi-si bilan bog’laning'",
       },
       {
-        title: 'BBC News O‘zbek TV dasturi',
-        url: '/uzbek/bbc_uzbek_tv/tv_programmes/w13xttqv?limit=4',
+        id: 'COOKIE_SETTINGS',
+        href: '#',
+        text: 'Do not share or sell my info',
+        lang: 'en-GB',
       },
     ],
+    copyrightText: 'BBC. Би-би-си ташқи сайтлар мазмуни учун масъул эмас.',
+  },
+  navigation: [
+    {
+      title: 'Бош саҳифа',
+      url: '/uzbek',
+    },
+    {
+      title: 'Ўзбекистон',
+      url: '/uzbek/topics/c8y949r98pgt',
+    },
+    {
+      title: 'Минтақа',
+      url: '/uzbek/topics/cwr9j9dz4gpt',
+    },
+    {
+      title: 'Дунё',
+      url: '/uzbek/topics/cl8l9mved19t',
+    },
+    {
+      title: 'Спорт',
+      url: '/uzbek/topics/cxnykykk1zkt',
+    },
+    {
+      title: 'Илм-Фан',
+      url: '/uzbek/topics/cg7262681krt',
+    },
+    {
+      title: 'Технология',
+      url: '/uzbek/topics/cjgn7n7v3yjt',
+    },
+    {
+      title: 'BBC News O‘zbek TV dasturi',
+      url: '/uzbek/bbc_uzbek_tv/tv_programmes/w13xttqv?limit=4',
+    },
+  ],
+};
+
+export const service: UzbekConfig = {
+  default: {
```

Comment:
Would be good to work out if there is a way to avoid this.

Reply:
I think we will need to keep this for historical reasons. Unless there is a nicer upstream method of redirecting non-variant links to `/cyr` then I think this is fine for backwards compatibility.